stream-chat-android-ui-uitests/src/androidTest/java/io/getstream/chat/android/uitests/ui/robot/compose/ComposeMessagesRobot.kt

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2014-2022 Stream.io Inc. All rights reserved.
+ *
+ * Licensed under the Stream License;
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://github.com/GetStream/stream-chat-android/blob/main/LICENSE
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.getstream.chat.android.uitests.ui.robot.compose
+
+import androidx.compose.ui.test.junit4.ComposeTestRule
+import androidx.compose.ui.test.onNodeWithContentDescription
+import androidx.compose.ui.test.performClick
+import androidx.compose.ui.test.performTextInput
+import org.junit.rules.TestRule
+
+/**
+ * A factory function for [ComposeMessagesRobot].
+ */
+internal fun composeMessagesRobot(
+    composeTestRule: ComposeTestRule,
```

Comment:
Could we potentially make these functions as extensions on the `ComposeTestRule`?

So that we can, in tests, say:

```
fun test() = with(composeTestRul) {
  typeMessageText {

  }

  ...
}
```

Instead of having to pass in the rule every time?

Reply:
Basically - make it more of a DSL

Reply:
Done